src/zk.py

```diff
@@ -225,18 +229,18 @@ def _acquire_lock(self, name, allow_queue, timeout, read_lock=False):
         if not acquired and self._release_lock_after_acquire_failed:
             logging.debug('Try to release and delete lock "%s", to recreate on next iter', name)
             try:
-                self.release_lock(name)
+                self.release_lock(name, host_description=host_description)
             except Exception:
                 for line in traceback.format_exc().split('\n'):
                     logging.error(line.rstrip())
         return acquired
 
-    def _get_lock(self, name, read_lock) -> Lock:
+    def _get_lock(self, name, read_lock, host_description=None) -> Lock:#
```

Comment:
`#` in the end of line, something strange...

Reply:
oops